### PR TITLE
vim-patch:partial:3f32a5f1601a

### PR DESCRIPTION
**Pull request by clason**
Update runtime files and translations
https://github.com/vim/vim/commit/3f32a5f1601ab2b0eba0caad00d4c26fb86a02a2

skip eval.txt (requires 8.2.4883)

### CODE REVIEW

File: runtime/doc/syntax.txt [hunk 1]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
 on the value of 'background'.  You can see the current settings with the
 ":highlight" command.
 							*hl-ColorColumn*
-ColorColumn	used for the columns set with 'colorcolumn'
+ColorColumn	Used for the columns set with 'colorcolumn'
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5099]:
```suggestion
ColorColumn	Used for the columns set with 'colorcolumn'.
```

File: runtime/doc/syntax.txt [hunk 2]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-Conceal*
-Conceal		placeholder characters substituted for concealed
+Conceal		Placeholder characters substituted for concealed
 		text (see 'conceallevel')
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5102]:
```suggestion
		text (see 'conceallevel').
```

File: runtime/doc/syntax.txt [hunk 3]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-CurSearch*
-CurSearch	used for highlighting a search pattern under the cursor
+CurSearch	Used for highlighting a search pattern under the cursor
 		(see 'hlsearch')
 							*hl-Cursor*
-Cursor		character under the cursor
-lCursor		the character under the cursor when |language-mapping|
+Cursor		Character under the cursor
+lCursor		The character under the cursor when |language-mapping|
 		is used (see 'guicursor')
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5109]:
```suggestion
Cursor		Character under the cursor.
lCursor		Character under the cursor when |language-mapping|
		is used (see 'guicursor').
```

File: runtime/doc/syntax.txt [hunk 4]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-CursorIM*
-CursorIM	like Cursor, but used when in IME mode |CursorIM|
+CursorIM	Like Cursor, but used when in IME mode |CursorIM|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5111]:
```suggestion
CursorIM	Like Cursor, but used when in IME mode. |CursorIM|
```

File: runtime/doc/syntax.txt [hunk 5]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-CursorColumn*
 CursorColumn	Screen-column at the cursor, when 'cursorcolumn' is set.
 							*hl-CursorLine*
 CursorLine	Screen-line at the cursor, when 'cursorline' is set.
 		Low-priority if foreground (ctermfg OR guifg) is not set.
 							*hl-Directory*
-Directory	directory names (and other special names in listings)
+Directory	Directory names (and other special names in listings)
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5118]:
```suggestion
Directory	Directory names (and other special names in listings).
```

File: runtime/doc/syntax.txt [hunk 6]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-DiffAdd*
-DiffAdd		diff mode: Added line |diff.txt|
+DiffAdd		Diff mode: Added line |diff.txt|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5120]:
```suggestion
DiffAdd		Diff mode: Added line. |diff.txt|
```

File: runtime/doc/syntax.txt [hunk 7]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-DiffChange*
-DiffChange	diff mode: Changed line |diff.txt|
+DiffChange	Diff mode: Changed line |diff.txt|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5122]:
```suggestion
DiffChange	Diff mode: Changed line. |diff.txt|
```

File: runtime/doc/syntax.txt [hunk 8]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-DiffDelete*
-DiffDelete	diff mode: Deleted line |diff.txt|
+DiffDelete	Diff mode: Deleted line |diff.txt|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5124]:
```suggestion
DiffDelete	Diff mode: Deleted line. |diff.txt|
```

File: runtime/doc/syntax.txt [hunk 9]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-DiffText*
-DiffText	diff mode: Changed text within a changed line |diff.txt|
+DiffText	Diff mode: Changed text within a changed line |diff.txt|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5126]:
```suggestion
DiffText	Diff mode: Changed text within a changed line. |diff.txt|
```

File: runtime/doc/syntax.txt [hunk 10]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 		By default, this is highlighted like |hl-NonText|.
 							*hl-TermCursor*
-TermCursor	cursor in a focused terminal
+TermCursor	Cursor in a focused terminal
 							*hl-TermCursorNC*
-TermCursorNC	cursor in an unfocused terminal
+TermCursorNC	Cursor in an unfocused terminal
 							*hl-ErrorMsg*
-ErrorMsg	error messages on the command line
+ErrorMsg	Error messages on the command line
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5135]:
```suggestion
ErrorMsg	Error messages on the command line.
```

File: runtime/doc/syntax.txt [hunk 11]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-WinSeparator*
-WinSeparator	separators between window splits
+WinSeparator	Separators between window splits
 							*hl-Folded*
-Folded		line used for closed folds
+Folded		Line used for closed folds
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5139]:
```suggestion
Folded		Line used for closed folds.
```

File: runtime/doc/syntax.txt [hunk 12]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-FoldColumn*
 FoldColumn	'foldcolumn'
 							*hl-SignColumn*
-SignColumn	column where |signs| are displayed
+SignColumn	Column where |signs| are displayed
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5143]:
```suggestion
SignColumn	Column where |signs| are displayed.
```

File: runtime/doc/syntax.txt [hunk 13]
```diff
@@ -5096,51 +5096,51 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
 							*hl-IncSearch*
 IncSearch	'incsearch' highlighting; also used for the text replaced with
 		":s///c"
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5146]:
```suggestion
		":s///c".
```

File: runtime/doc/syntax.txt [hunk 14]
```diff
@@ -5181,7 +5181,7 @@ NonText		'@' at the end of the window, characters from 'showbreak'
 		(e.g., ">" displayed when a double-wide character doesn't
 		fit at the end of the line). See also |hl-EndOfBuffer|.
 							*hl-Normal*
-Normal		normal text
+Normal		Normal text
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5184]:
```suggestion
Normal		Normal text.
```

**Review by zeertzjq — COMMENTED** [review]
Missing too many periods. Also I think there are too many skipped changes for this to be a full port.

**Comment by clason** [comment]
Yes, sorry, I missed that periods were added; I'll fix that ASAP.

But I don't get the "too many skipped changes" -- translations are N/A, so shouldn't count?

**Comment by zeertzjq** [comment]
I mean eval.txt

**Comment by clason** [comment]
OK; will rename when squashing.

File: runtime/doc/syntax.txt [hunk 15]
```diff
@@ -5096,56 +5096,56 @@ These are the builtin highlighting groups.  Note that the highlighting depends
...
-CurSearch	used for highlighting a search pattern under the cursor
-		(see 'hlsearch')
+CurSearch	Used for highlighting a search pattern under the cursor
+		(see 'hlsearch').
 							*hl-Cursor*
-Cursor		character under the cursor
-lCursor		the character under the cursor when |language-mapping|
-		is used (see 'guicursor')
+Cursor		Character under the cursor.
+lCursor		The character under the cursor when |language-mapping|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5108]:
```suggestion
lCursor		Character under the cursor when |language-mapping|
```

File: runtime/doc/syntax.txt [hunk 16]
```diff
@@ -5181,11 +5181,11 @@ NonText		'@' at the end of the window, characters from 'showbreak'
...
 							*hl-Normal*
-Normal		normal text
+Normal		Normal text.
 							*hl-NormalFloat*
 NormalFloat	Normal text in floating windows.
 							*hl-NormalNC*
-NormalNC	normal text in non-current windows
+NormalNC	Normal text in non-current windows.
 							*hl-Pmenu*
 Pmenu		Popup menu: normal item.
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5190]:
```suggestion
Pmenu		Popup menu: Normal item.
```

File: runtime/doc/syntax.txt [hunk 17]
```diff
@@ -5168,11 +5168,11 @@ MatchParen	The character under the cursor or just before it, if it
 		is a paired bracket, and its match. |pi_paren.txt|
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5168]: Missing `The character` -> `Character`

File: runtime/doc/syntax.txt [hunk 18]
```diff
@@ -5195,7 +5195,7 @@ PmenuSbar	Popup menu: scrollbar.
 							*hl-PmenuThumb*
```
Comment (zeertzjq) [inline runtime/doc/syntax.txt:5195]: Missing capitalization above

File: runtime/doc/syntax.txt [hunk 19]
```diff
@@ -5220,30 +5220,30 @@ SpellRare	Word that is recognized by the spellchecker as one that is
 		hardly ever used. |spell|
 		Combined with the highlighting used otherwise.
 							*hl-StatusLine*
-StatusLine	status line of current window
+StatusLine	Status line of current window.
 							*hl-StatusLineNC*
-StatusLineNC	status lines of not-current windows
-		Note: if this is equal to "StatusLine" Vim will use "^^^" in
+StatusLineNC	Status lines of not-current windows.
+		Note: If this is equal to "StatusLine" Vim will use "^^^" in
```
Comment (seandewar) [inline runtime/doc/syntax.txt:5226]:
```suggestion
		Note: If this is equal to "StatusLine", Vim will use "^^^" in
```

**Review by seandewar — COMMENTED** [review]
`:for` over strings is supported, so we could manually cherry-pick https://github.com/vim/vim/commit/3f32a5f1601ab2b0eba0caad00d4c26fb86a02a2#diff-b3aeef96275167ddcc1e5fee6d51b8f2018f9359f921d86ad6e7035ca80f20e9R3549-R3550 (including the empty line, but without E1177).

Apart from the other comment, LGTM.

**Review by seandewar — APPROVED** [review]
otherwise LGTM

File: runtime/doc/eval.txt [hunk 20]
```diff
@@ -3141,10 +3141,12 @@ text...
 			a |Blob| or a |String|.
 
 			Variable {var} is set to the value of each
-			item.  When an error is detected for a command inside
-			the loop, execution continues after the `endfor`.
-			Changing {object} inside the loop affects what items
-			are used.  Make a copy if this is unwanted: >
+			item.
```
Comment (seandewar) [inline runtime/doc/eval.txt:3144]: Keep `item.` on the same line as above.

File: runtime/doc/eval.txt [hunk 21]
```diff
@@ -3137,12 +3137,16 @@ text...
...
+			each item in {object}.  {object} can be a |List|,
+			a |Blob| or a |String|.
+
+			Variable {var} is set to the value of each
+			item.
+
+			When an error is detected for a command inside the
+			loop, execution continues after the `endfor`. Changing
+			{object} inside the loop affects what items are used.
+			Make a copy if this is unwanted: >
```
Comment (seandewar) [inline runtime/doc/eval.txt:3149]: Oh, and format this like:
```suggestion
			When an error is detected for a command inside the
			loop, execution continues after the `endfor`.
			Changing {object} inside the loop affects what items
			are used. Make a copy if this is unwanted: >
```

Comment (clason) [reply runtime/doc/eval.txt:3149]: I just did `gq` and was happy it looked so nice...

**Review by seandewar — APPROVED** [review]

**Comment by zeertzjq** [comment]
`text` was changed to `Text` in the patch, although the docs here has already diverged.

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**